src/styles/base.scss

```diff
@@ -124,11 +124,11 @@ body {
 
 /*Font weights*/
 
-.fw-light {
+.fw-light, .w-300 {
   font-weight: $fw-light;
 }
 
-.fw-regular {
+.fw-regular, .w-400 {
```

Comment:
@sanketbansal Why are you adding this as duplicate names for using this font?, instead use `fw-light` or `fw-regular` in the places where you have used it

Reply:
Because now I have to search these styles in each and every PR I have made and made the insignificant change.